kong-0.2.0-1.rockspec

```diff
@@ -1,9 +1,9 @@
 package = "kong"
-version = "0.1.1beta-2"
+version = "0.2.0-1"
 supported_platforms = {"linux", "macosx"}
 source = {
   url = "git://github.com/Mashape/kong",
-  tag = "0.1.1beta-2"
```

Comment:
Mhh yeah. You should specify we are using semver versioning somewhere then. I was talking about bumping the version when we are actually releasing a version only. This doesn't change the behaviour of the project, it only refactors. Doesn't qualify for a release.